provider/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptBasePlugin.kt

```diff
@@ -24,16 +24,14 @@ import org.gradle.kotlin.dsl.accessors.tasks.UpdateProjectSchema
 import org.gradle.kotlin.dsl.apply
 import org.gradle.kotlin.dsl.task
 
-
 class KotlinScriptBasePlugin : Plugin<Project> {
-    override fun apply(project: Project): Unit =
+    override fun apply(project: Project) : Unit = // ktlint-disable
```

Comment:
I discovered a small bug:
https://github.com/shyiko/ktlint/issues/77

Reply:
❌ This PR should be updated to use 0.9.2 that fixes this bug before being merged.